### PR TITLE
Feature/ddw 5 Send bug reports with logs from Daedalus

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces a feature for sending bug reports with logs from Daedalus.
Also, maximum log size for log rotation is set to 20 megabytes.

**Tasks**
- [x] Connect with API (PR: https://github.com/input-output-hk/cardano-report-server/pull/12)
- [x] Inject missing Japanese translations
- [x] Inject missing Japanese translations for reporting an issue from the connecting/syncing screens
- [x] Dark-blue and cardano elements color on reporting an issue from the connecting/syncing screens
- [x] Update code after Daedalus logs have been moved into Pub directory: https://github.com/input-output-hk/daedalus/pull/505
- [x] Replace hardcoded report server URL with `process.env.REPORT_URL` variable which will be set at webpack build time. The value will be URL with appended slash like `https://...:8080/`

screen shot 2018-01-26 at 16 56 43

report issue from connecting screen

report issue from syncing screen

### CODE REVIEW

**Comment by DominikGuzei** [comment]
Testing this now!

File: app/api/ada/sendAdaBugReport.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    path: '/report',
+    port: 8000,
+  }, {
+    application,
+    version: '0.0.1',
+    build: environment.build,
+    os: platform,
+    logs,
+    date: moment().format('YYYY-MM-DDTHH:mm:ss'),
+    magic: 2000000000,
```
[the remark this answers is not in the document]
Comment (wiatrM) [reply app/api/ada/sendAdaBugReport.js]: https://www.youtube.com/watch?v=NFQCYpIHLNQ

[the remark this answers is not in the document]
Comment (nikolaglumac) [reply app/api/ada/sendAdaBugReport.js]: @DominikGuzei if you are asking about `magic` - this is how we were told to call this endpoint. We have no details on the reasons...

**Comment by tomothespian** [comment]
@DominikGuzei  button color improvements.

Hover:
hover

Active:
active

**Comment by nikolaglumac** [comment]
@DominikGuzei we are ready for another review now :)

**Comment by tomothespian** [comment]
alternative bug report send

**Comment by nikolaglumac** [comment]
Great work @tomothespian 👍 
I have reviewed the new manual-bug-report dialog and it works great 🎉 
cc @darko-mijic

**Comment by darko-mijic** [comment]
Thanks, guys! @tomothespian this was really quick. Appreciate it!

**Comment by nikolaglumac** [comment]
@tomothespian @darko-mijic Japanese translations added:

screen shot 2018-02-13 at 22 42 52

**Comment by nikolaglumac** [comment]
@tomothespian I have added one last TODO in the PR's description. The goal is to replace hardcoded report server URL with the one we will get via ENV variable. After this part is done we are ready for the final review and merge.
cc @darko-mijic @DominikGuzei

**Comment by nikolaglumac** [comment]
@DominikGuzei @darko-mijic we have the REPORT_URL env variable and the default `staging` report server URL implemented 🎉  https://github.com/input-output-hk/daedalus/pull/691/commits/fb2ecda1d1ebe17d713ff819d3033fb4cf077301

This means we are officially ready for the review and merging so please don't be shy and join me in reviewing this masterpiece of a feature 💯 

**NOTE**
Current `staging` report-server hasn't been updated which means the `/reports` endpoint is not yet operational. @domenkozar please inform us once this `/reports` update is deployed. Thanks!

**Comment by domenkozar** [comment]
cc @cleverca22 who's working on the installer part.

**Comment by domenkozar** [comment]
See https://github.com/input-output-hk/daedalus/pull/724/files

**Comment by nikolaglumac** [comment]
Thanks @domenkozar!
One last question - can you please tell us when will the `http://report-server.awstest.iohkdev.io:8080/report` endpoint be operational? We are eager to test it...

**Comment by domenkozar** [comment]
Stuck on https://github.com/input-output-hk/cardano-report-server/pull/12#issuecomment-365625058, after it's in release branch we can deploy it

**Comment by nikolaglumac** [comment]
@domenkozar OK. Would be cool if you could ping us once this update is deployed. Thanks!

**Comment by DominikGuzei** [comment]
This PR was tested with latest staging reporting-server, will merge now 🎉

**Pull request merged**